cloudify_cli/cli/cfy.py

```diff
@@ -803,6 +803,13 @@ def __init__(self):
             help=helptexts.SKIP_PLUGINS_VALIDATION
         )
 
+        self.skip_plugins_upload_validation = click.option(
+            '--skip-validation',
```

Comment:
this only affects local plugins - maybe it should be reflected in the flag name?

Reply:
`--skip-local-plugins-validation`? would be too long.
I chose a flag name that makes sense to the user.

Reply:
However, the help text should probably be adjusted. Will fix.